balancer-js/examples/flashSwap.ts

```diff
@@ -0,0 +1,88 @@
+import Web3 from 'web3';
+import dotenv from 'dotenv';
+import { BalancerSDK, Network, SwapType } from '../src';
+import { BalancerSdkConfig } from '../src/types';
+import { BALANCER_NETWORK_CONFIG } from '../src/lib/constants/config';
+
+dotenv.config();
+
+const { WALLET_PRIVATE_KEY, INFURA } = process.env;
```

Comment:
It would be nice if this were called `INFURA_KEY` or something more descriptive.

Reply:
Feel free to change it throughout.

Reply:
https://github.com/balancer-labs/balancer-sdk/pull/45